packages/atomic/src/components/search/atomic-result-template/atomic-result-template.mdx

```diff
@@ -0,0 +1,117 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import * as AtomicResultTemplateStories from './atomic-result-template.new.stories';
+import { AtomicDocTemplate } from '../../../../storybook-utils/documentation/atomic-doc-template';
+
+
+<Meta of={AtomicResultTemplateStories} />
+
+<AtomicDocTemplate
+  stories={AtomicResultTemplateStories}
+  defaultStory="Default"
+  githubPath="search/atomic-result-template/atomic-result-template.ts"
+  tagName="atomic-result-template"
+  className="AtomicResultTemplate"
+>
+
+This component defines the UI display of your search results. 
```

Comment:
As discussed in person, ensure that this section doesn't duplicate the JSDoc and follows established patterns for "Usage" documentation.

Also apply the corrections to the atomic-product-tempalte mdx